avocado/utils/nvme.py

```
@@ -186,6 +186,11 @@ def is_ns_exists(controller_name, ns_id):
     ns_list = get_current_ns_ids(controller_name)
     if ns_id in ns_list:
         return True
+    else:
+        for ctrl in get_alternate_controller_name(controller_name):
```

Ruff (0.11.9)

Loop control variable `ctrl` not used within loop body

Rename unused `ctrl` to `_ctrl`

(B007)